tests/plugins/wysiwygarea/detached.js

```diff
@@ -33,7 +33,9 @@
 				} );
 
 				editorContainer.remove();
-				editorContainerParent.append( editorContainer );
+				setTimeout( function() {
+					editorContainerParent.append( editorContainer );
+				}, 200 );
```

Comment:
Why do we need a `setTimeout` here?

Reply:
That was a fix for unstable test: https://github.com/ckeditor/ckeditor4/pull/5023#issuecomment-1015156203 - however, I reverted it and FF seems to be fine... I try to restart CI a few times and we will see

Reply:
And yet, the test fails again randomly... It is because:
https://github.com/ckeditor/ckeditor4/blob/ee3949d19b95af2eef96ec561e647b2eb450bd64/plugins/wysiwygarea/plugin.js#L164-L177

- ☝️ line 170 leads to https://github.com/ckeditor/ckeditor4/blob/ee3949d19b95af2eef96ec561e647b2eb450bd64/core/editable.js#L555

- line 176 leads to document loads again and lead to the https://github.com/ckeditor/ckeditor4/blob/ee3949d19b95af2eef96ec561e647b2eb450bd64/plugins/wysiwygarea/plugin.js#L119

The test fails when the document did load before the assertion - and that sometimes happened before the resume is executed. But since I wrote all of this - I figure out - that simply getting selection before the `resume` is executed is just fine.